source/common/network/cidr_range.cc

```diff
@@ -55,6 +55,13 @@ bool CidrRange::operator==(const CidrRange& other) const {
   }
 }
 
+const Ip* CidrRange::ip() const {
+  if (address_ != nullptr) {
+    return address_->ip();
+  }
+  return nullptr;
+}
+
 const Ipv4* CidrRange::ipv4() const {
```

Comment:
Do we still need this method?

Reply:
Good catch.  I'll push a version that removes this and a couple other methods that just mirror the interface of Ip.